ADOPTERS.md

```diff
@@ -14,7 +14,7 @@ Below are the adopters of project Spark Operator. If you are using Spark Operato
 | CloudZone | @iftachsc | Evaluation | Big Data Analytics Consultancy |
 | Cyren | @avnerl | Evaluation | Data pipelines |
 | [C2FO](https://www.c2fo.com/) | @vanhoale | Production | Data Platform / Data Infrastructure |
-| [Data Mechanics](https://www.datamechanics.co)  | @jrj-d | Production | Managed Spark Platform |
+| [Spot by Netapp](https://spot.io/product/ocean-apache-spark/)  | @ImpSy | Production | Managed Spark Platform |
```

Comment:
I am wondering why is Data Mechanics deleted? It would be better to raise a dedicated PR to add your company to adopters list.

Reply:
Sorry I thought it was clear in the commit msg

Datamechanics has been acquired 3 years ago and the last remaining project has been discontinued in May

The new join product lives under Spot by Netapp

Reply:
Got it. Would you like to raise a dedicated PR to replace Data Mechanics with Sprt by Netapp?

Reply:
I've created a PR for this -> https://github.com/kubeflow/spark-operator/pull/2168